<launch>
    <arg name='tracking_mode' default='track'/>     <!-- 'track', 'hover', 'home' -->
    <arg name='yaw_command_mode' default='yawrate'/>     <!-- 'yaw', 'yawrate' -->
    <node name='mav_nmpc_tracker_node' pkg='mav_nmpc_tracker' type='nmpc_tracker_node_square_thrust.py' output='screen'>
        <rosparam file="$(find mav_nmpc_tracker)/config/nmpc_tracker_square_thrust.yaml" />
        <param name='tracking_mode' value='$(arg tracking_mode)'/>
        <param name='yaw_command_mode' value='$(arg yaw_command_mode)'/>
        <!-- <remap from="/mav_odometry" to="/mavros/local_position/odom_local"/> -->
        <!-- <remap from="/mav_trajectory" to="/command/trajectory"/> -->
    </node>
</launch>